test/e2e/framework/deployment/wrappers.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
I would prefer to collapse the files in here with `util.go` (I also don't love `util.go` as a name but that can probably be done in a follow-up PR) because aside from the fact that the methods here are just wrappers for `testutils`, the methods are still similar to what's in `util.go`. What do you think?

Reply:
That makes sense. Would it be better to separate the file into public and private functions?
and change the file names to `helpers.go` and `deployment.go`?

Reply:
Which methods are private?

Reply:
Made the changes on the latest commit to see it better.
The couple functions in `helpers.go` are not used anywhere outside of `framework/deployment`.
`logReplicaSetsOfDeployment` and `logPodsOfDeployment` are private, while `MakeDeployment` is not used outside of the package.

Reply:
hmm 🤔. I'd like to avoid files like `helper.go` that become a dumping ground for generic functions in the package. How about we put all the create/update deployment functions in `fixtures.go` (including MakeDeployment) and all the `Wait*` methods in `wait.go`? The log methods can go in the same file as the callers. What do you think?

Reply:
I like that idea. et me give it a try